libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -19,6 +19,18 @@
         <mat-icon>remove_circle</mat-icon>
       </button>
     </div>
+    <div class="finished-text" *ngIf="readingItem.finished">
+      Finished on: {{ readingItem.finishedDate | date:'MM/dd/yyyy' }}
+    </div>
+    <button
+      mat-icon-button
+      class="mark-finish-text"
+      [attr.aria-label]="'Mark ' + readingItem.title + ' as read'"
+      *ngIf="!readingItem.finished"
+      (click)="markAsFinished(readingItem)"
+    >
+      Mark as read <mat-icon class="mark-finish-circle">check_circle</mat-icon>
```

Comment:
You've got a label here _and_ the aria label. Do you need both?

Reply:
Removed the aria label